contrib/scripts/verifier_diff.py

```diff
@@ -54,22 +54,28 @@ def organize_data(data) -> dict:
     """
     organized = {}
     for entry in data:
-        key = (entry["collection"], entry["build"],
-               entry["load"], entry["program"])
-        organized[key] = entry["insns_processed"]
+        if not key in entry:
+            logging.error(f"Key '{key}' not found in data.")
+            break
+        if not entry[key].isnumeric():
```

Comment:
Running this locally I get:
```
Traceback (most recent call last):
  File "verifier_diff.py", line 214, in <module>
    main()
    ~~~~^^
  File "verifier_diff.py", line 210, in main
    plot_comparison(args.file_before, args.file_after, args.key, output_dir)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "verifier_diff.py", line 77, in plot_comparison
    data1 = organize_data(load_json(file1), key)
  File "verifier_diff.py", line 60, in organize_data
    if not entry[key].isnumeric():
           ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'int' object has no attribute 'isnumeric'
```

We can probably do:
```suggestion
        if not isinstance(entry[key], (int, float)) or not str(entry[key]).isnumeric():
```

Reply:
Uh, weird. I don't get that error locally. Maybe different Python versions?

Reply:
I'm using `Python 3.13.3` from `apt` 🤔

Reply:
3.12.3 here :shrug: